zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServerMain.java

```diff
@@ -239,7 +239,28 @@ ServerCnxnFactory getCnxnFactory() {
 
     // VisibleForTesting
     ServerCnxnFactory getSecureCnxnFactory() {
-        return secureCnxnFactory;
+    }
```

Comment:
The `return secureCnxnFactory;` somehow got swallowed here, causing CI errors.

Reply:
@ztzg I made  mess while resolving conflicts using the github UI :-) 
I have fixed the code